### PR TITLE
attributes on data members are supported.

### PR DESCRIPTION
**Pull request by klemens-morgenstern**

### CODE REVIEW

File: source/api/AST/Serializer.cpp [hunk 1]
```diff
@@ -445,6 +445,24 @@ getMemberTypeInfo(
 {
     Assert(D && "Expect non-null FieldDecl in getMemberTypeInfo");
     parseJavadoc(I.javadoc, D);
+
+    for (auto attr : D->attrs())
+    {
+        switch (attr->getKind())
+        {
+            case clang::attr::Kind::NoUniqueAddress:
+                I.Attributes.set<FieldAttributes::hasNoUniqueAddress>();
```
Comment (vinniefalco) [inline source/api/AST/Serializer.cpp:454]: "Attributes" why so wordy

**Comment by klemens-morgenstern** [comment]
```cpp
D:\a\mrdox\mrdox\test-files\old-tests\no_unique_address.cpp:8:7: error: unknown attribute 'no_unique_address' ignored [-Werror,-Wunknown-attributes]
    [[no_unique_address, deprecated, maybe_unused]] Empty e = Empty{};
```

Looks like this is a clang on windows issue

File: include/mrdox/Metadata/FieldType.hpp [hunk 2]
```diff
@@ -14,12 +14,22 @@
 
 #include <mrdox/Platform.hpp>
 #include <mrdox/Metadata/Type.hpp>
+#include <mrdox/Metadata/BitField.hpp>
```
Comment (vinniefalco) [inline include/mrdox/Metadata/FieldType.hpp:17]: alphabetical order?

File: include/mrdox/Metadata/FieldType.hpp [hunk 3]
```diff
@@ -31,31 +41,23 @@ struct FieldTypeInfo
...
         , DefaultValue(DefaultValue)
+        , Flags(Flags)
     {
     }
-
-#if 0
-    // VFALCO What was this for?
-    bool
-    operator==(
-        FieldTypeInfo const& Other) const
```
Comment (vinniefalco) [inline include/mrdox/Metadata/FieldType.hpp:52]: The reason I leave these things in there for now is so I don't forget

**Pull request merged**